src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Couchbase/Instrumentation.xml

```diff
@@ -7,20 +7,7 @@ SPDX-License-Identifier: Apache-2.0
 	<instrumentation>
 
 		<tracerFactory>
-			<match assemblyName="Couchbase" className="Couchbase.CouchbaseClient">
-				<exactMethodMatcher methodName="ExecuteRemove" parameters="System.String,Couchbase.Operations.PersistTo,Couchbase.Operations.ReplicateTo" />
-				<exactMethodMatcher methodName="Get" parameters="Couchbase.IView" />
-				<exactMethodMatcher methodName="PerformConcatenate"/>
-				<exactMethodMatcher methodName="PerformMutate"/>
-				<exactMethodMatcher methodName="PerformRemove"/>
-				<exactMethodMatcher methodName="PerformStore" />
-				<exactMethodMatcher methodName="PerformTryGet" />
-				<exactMethodMatcher methodName="PerformTryGetAndTouch"/>
-				<exactMethodMatcher methodName="PerformTryGetWithLock"/>
-				<exactMethodMatcher methodName="Touch" />
-				<exactMethodMatcher methodName="Unlock"/>
-			</match>
```

Comment:
Why was this removed? Does removing this break the old instrumentation?

Reply:
It was removed because it's unused. 

For reasons that long pre-date me, the `CanWrap()` methods in the Couchbase wrappers are looking for specific class and method names, and none of them are looking for `Couchbase.CouchbaseClient`.